dm/tests/many_tables/run.sh

```diff
@@ -42,6 +42,13 @@ function run() {
 	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
 
 	dmctl_start_task_standalone
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"\"totalTables\": \"500\"" 1 \
+		"\"completedTables\"" 1 \
```

Comment:
not sure if this load progress finished quickly, the test will never passed (completedTable >1) , so i perfer to only check `"\"totalTables\": \"500\"" 1 \`

Reply:
this is test dump progress. And "only check "\"totalTables\": \"500\"" 1 \" doesn't help if dump unit is passed

Reply:
`totalTables` is in `DumpStatus`  🤔

https://github.com/pingcap/tiflow/blob/c2b9e858393a15b446e54fa473f5c4988b031831/dm/dm/proto/dmworker.proto#L104

Reply:
please check PR description before you review

Reply:
sorry... `"\"completedTables\"" 1 \` is not means there is one completed tables... it means on record from dmctl query-status result